src/Sylius/Component/Attribute/Model/Attribute.php

```diff
@@ -44,6 +44,9 @@ class Attribute implements AttributeInterface
     /** @var int */
     protected $position;
 
+    /** @var bool */
+    protected $translatable = false;
```

Comment:
👍  I was thinking about changing it to  `translatableValue `

Reply:
I would be for `$translatable`, it describes more clearly that it will be a boolean value in my opinion, and it is more consistent with methods as `isTranslatable`